start.sh

```diff
@@ -8,8 +10,12 @@ echo "date.timezone = ${TZ}" >> /etc/php.ini
 sed -i "s/^\(memory_limit =\).*/\1 ${PHP_MEMORY_LIMIT}/" /etc/php.ini
 sed -i "s/^\(max_execution_time =\).*/\1 ${PHP_MAX_EXECUTION_TIME}/" /etc/php.ini
 
-rm /etc/localtime
-ln -s /usr/share/zoneinfo/${TZ} /etc/localtime
+if [ ! -z "${TZ}" ]; then
+  rm /etc/localtime
+  ln -s /usr/share/zoneinfo/${TZ} /etc/localtime
+else
+  echo "Using default timezone sinze TZ is unset."
```

Comment:
Mind correcting the spelling of "sinze"-->"since"? Everything else looks good. I should do a spell check myself, given I know there are likely several others to clean (^_^)

Reply:
ah good catch :)